src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+.mainheader {
```

```suggestion
.mainHeader {
```
For this task it's not necessary to use classes, because all tags have the same styling, but if you want to use classes - name them properly (in camelcase - `mainHeader`, `articleHeader` etc)